content/cloud/variables.md

```diff
@@ -0,0 +1,205 @@
+title = "Configuring Spin Application Variables and Secrets"
+date = "2023-06-05T16:40:01.672734Z"
+template = "cloud_main"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/variables.md"
+
+---
+
+- [Prerequisites](#prerequisites)
+- [Creating Our Spin Application](#creating-our-spin-application)
+- [Adding Variables to an Application Component](#adding-variables-to-an-application-component)
+- [Using Variables in a Spin Application](#using-variables-in-a-spin-application)
+- [Deploying the Application to Fermyon Cloud](#deploying-the-application-to-fermyon-cloud)
+
+Spin supports dynamic application variables. Instead of being static, their values can be updated without modifying the application, creating a simpler experience for rotating secrets, updating API endpoints, and more. 
+
+These variables are defined in a Spin application manifest (in the `[variables]` section) and are provided by a configuration provider. When using Spin locally, the configuration provider can be Vault for secrets or host environment variables. You can refer to the [dynamic configuration documentation](../spin/dynamic-configuration.md) to learn how to configure variables locally. In Fermyon Cloud, you can set and update variables for Spin applications using the [`spin cloud variables`](./cloud-plugin.md#variables) command.
```

Comment:
The pathing on the two links here looks a bit odd to my eyes.  In other paragraphs (e.g. line 24) you use absolute paths, which I believe is the general preference... probably good to be consistent anyway